### PR TITLE
Add types.ts and user interface

### PR DESCRIPTION
**Pull request by zaviermiller**
Added `types.ts` file and the `User` interface. Wasn't sure if I was supposed to create the model yet since mongo isn't connected, but can/will if needed

### CODE REVIEW

File: utils/types.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ObjectID } from "mongodb";
+
+// Keep these in sync with the backend schema
+
+export interface User {
+  _id?: ObjectID;
+  email?: string;
+  password?: string;
```
Comment (kemalfidan) [inline utils/types.ts]: Can you remove the password field here? To let users quickly register for events, KKB prefers to not create accounts - so we won't need a password.

**Review by kemalfidan — COMMENTED** [review]
Good first PR for the repo ;). I made 1 comment about the password field but otherwise ship it.

**Comment by kemalfidan** [comment]
Also can you tag Selena and the other devs to this PR as well? Vishal is with a different project so I'm not sure if he has time to review our PRs. Thanks!

**Review by rossketron — DISMISSED** [review]
Looks good to me!

**Review by andlrutt — DISMISSED** [review]
Looks great! Went ahead and approved, ofc pending the removal of the password as per Kemal's comment

**Review by rossketron — APPROVED** [review]

**Review by andlrutt — APPROVED** [review]
looks good!

**Review by kemalfidan — APPROVED** [review]

**Pull request merged**